taskana-adapter-camunda-spring-boot-test/src/test/java/pro/taskana/adapter/integration/TestTaskAcquisition.java

```diff
@@ -268,7 +267,8 @@ private void retrieveTaskanaTaskAndVerifyTaskVariables(
       Assert.assertThat(assumedVariablesString, SameJSONAs.sameJSONAs(taskanaVariablesString));
 
     } catch (TaskNotFoundException | NotAuthorizedException e) {
-      LOGGER.info("Caught {}, while trying to create a taskana task and verify its variables", e, e);
+      LOGGER.info(
+          "Caught {}, while trying to create a taskana task and verify its variables", e, e);
```

Comment:
If the message contains curly braces, there should be one replacement parameter: e.getClass().getName()